app/Http/Controllers/DiagnosticsController.php

```diff
@@ -366,12 +364,11 @@ public function get_config()
 	}
 
 	/**
-	 * This function return the Diagnostic data as an JSON array.
-	 * should be used for AJAX request.
+	 * Return the requested information.
 	 *
 	 * @return array
 	 */
-	public function get()
+	private function get_data()
 	{
 		$errors = $this->get_errors();
 		$infos = ['You must be logged to see this.'];
```

Comment:
Not really in scope for the PR, but it feels like an `else` would be cleaner here?